url.bs

```diff
@@ -1561,11 +1563,30 @@ string <var>input</var>, optionally with a <a>base URL</a> <var>base</var>, opti
 
    <dt><dfn>special authority ignore slashes state</dfn>
    <dd>
-    <p>If <a>c</a> is neither "<code>/</code>" nor "<code>\</code>", set <var>state</var>
-    to <a>authority state</a>, and decrease <var>pointer</var> by one.
-
-    <p>Otherwise, <a>syntax violation</a>.
+    <ol>
+      <li>
+      <p>If <a>c</a> is neither "<code>/</code>" nor "<code>\</code>", run these substeps:
+        <ol>
+          <li>
+            <p>If <var>url</var>'s <a for=url>scheme</a> is "<code>file</code>", run these
+            subsubsteps:
+
+            <ol>
+             <li><p>If <a>remaining</a> does not start with "<code>//</code>",
+             <a>syntax violation</a>.
+
+             <li><p>Set <var>state</var> to <a>file state</a>.
+            </ol>
+          <li>
+            <p>Otherwise, set <var>state</var>
+            to <a>authority state</a>, and decrease <var>pointer</var> by one.</p>
```

Comment:
Running the jsdom/whatwg-url tests, this pointer decrease needs to happen even if the scheme is file.